django/db/backends/postgresql/introspection.py

```diff
@@ -56,18 +56,27 @@ def get_table_description(self, cursor, table_name):
         Return a description of the table with the DB-API cursor.description
         interface.
         """
-        # As cursor.description does not return reliably the nullable property,
-        # we have to query the information_schema (#7783)
+        # As cursor.description does not reliably return the nullable property
+        # (#7783) and information_schema does not contain details of
+        # materialized views (#29722), we have to query the pg_catalog tables.
```

Comment:
I don't think the ticket references are needed. Also, you could rewrite without "we" as per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style), e.g. “Loop over” rather than “We loop over”.

Unrelated: not sure if this commit makes sense on its own?

Reply:
Ok. Will tweak the comment and merge the commit with the one following it.